src/lib/device-types/index.ts

```diff
@@ -307,16 +331,20 @@ export const getImageVersions = (slug: string): Promise<ImageVersions> => {
 			return Promise.props({
 				buildId,
 				ignored: getIsIgnored(normalizedSlug, buildId),
-				hasDeviceTypeJson: getDeviceTypeJson(
-					normalizedSlug,
-					buildId,
-				).catchReturn(null),
-			});
+				hasDeviceTypeJson: getDeviceTypeJson(normalizedSlug, buildId),
+			}).catchReturn(null);
```

Comment:
That's probably an issue with the bluebird typings, since I was (and still) getting `expected 1-6 arguments, but got 0.`. Otherwise we would have to go with `.catch(_.noop)` which doesn't make it any better...

Reply:
Ok, `.catchReturn(undefined)` is slightly better then, because that's the equivalent to `.catchReturn()`